core/frontend/src/RealityDataSource.ts

```diff
@@ -181,16 +180,7 @@ class RealityDataSourceImpl implements RealityDataSource {
    * There will aways be only one reality data connection for a corresponding reality data source key.
```

Comment:
Comment is no longer true, right?

Reply:
Comment updated

Reply:
@pmconne To test this bug will required to add an integration test that connect to context share. I haven't found integration test that do that. This will required a little more thinking and work and I think should be part of another PR. I will sent you a private email. In the meantime, could you aprrove this PR.